app/lib/navigation.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:app/screens/history/history.dart';
+import 'package:app/screens/in_progress/in_progress.dart';
+import 'package:flutter/material.dart';
+
+class NavigationBarController extends StatefulWidget {
+  @override
+  _NavigationBarControllerState createState() =>
+      _NavigationBarControllerState();
+}
+
+class _NavigationBarControllerState extends State<NavigationBarController> {
+  int _selectedIndex = 0;
+  final List<Widget> _widgetOptions = <Widget>[
+    InProgress(),
+    TravelHistory(),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Center(
+        child: _widgetOptions.elementAt(_selectedIndex),
+      ),
+      bottomNavigationBar: BottomNavigationBar(
+        items: [
+          BottomNavigationBarItem(
```

Comment:
Could slap some TODOs in here for when the time comes

Reply:
Will do in my next commit